torch/fx/graph.py

```diff
@@ -243,10 +250,54 @@ def format_node(n : Node) -> Optional[str]:
         for node_str in node_strs:
             if node_str:
                 s += '\n    ' + node_str
-        if self.result:
+        if hasattr(self, 'result'):
```

Comment:
hm, why hasattr? Why not to make it just None in the constructor?

Reply:
idk, this is just how it was. We may be blasting this away in the future, so I'd prefer not to change it here